nncf/experimental/common/tensor_statistics/collectors.py

```diff
@@ -31,6 +31,13 @@
 AggregationAxes = Tuple[int, ...]
 
 
+class OutputMetadata:
```

Comment:
Could you briefly explain why you need to introduce the  OutputMetadata abstraction?

Reply:
@daniil-lyakhov

Reply:
OV backend is using port_id and node name to specify unique output. ONNX, in other hand, is using only edge name to specify this. As this output identificatory is backend-specific, I've introduced backend-specific output metadata class

Reply:
reverted